lib/auth/native/native.go

```diff
@@ -310,6 +310,12 @@ func (k *Keygen) GenerateUserCert(c services.UserCertParams) ([]byte, error) {
 	return ssh.MarshalAuthorizedKey(cert), nil
 }
 
+const (
+	principalLocalhost  = "localhost"
+	principalLoopbackV4 = "127.0.0.1"
+	principalLoopbackV6 = "::1"
+)
```

Comment:
We have a constants file: https://github.com/gravitational/teleport/blob/master/constants.go

Reply:
Moved.
Also switched to a typed constant. Would be nice to use those more instead of opaque strings everywhere (for readability and type safety reasons)